airflow/contrib/auth/backends/google_auth.py

```diff
@@ -16,176 +16,176 @@
+#                                     'google_oauth_callback',
+#                                     self.oauth_callback)
+#
+#     def login(self, request):
+#         log.debug('Redirecting user to Google login')
+#         return self.google_oauth.authorize(callback=url_for(
+#             'google_oauth_callback',
+#             _external=True),
+#             state=request.args.get('next') or request.referrer or None)
+#
+#     def get_google_user_profile_info(self, google_token):
+#         resp = self.google_oauth.get(
+#             'https://www.googleapis.com/oauth2/v1/userinfo',
+#             token=(google_token, ''))
+#
+#         if not resp or resp.status != 200:
+#             raise AuthenticationError(
+#                 'Failed to fetch user profile, status ({0})'.format(
+#                     resp.status if resp else 'None'))
+#
+#         return resp.data['name'], resp.data['email']
+#
+#     def domain_check(self, email):
+#         domain = email.split('@')[1]
+#         domains = _get_config_param('domain').split(',')
+#         if domain in domains:
+#             return True
+#         return False
+#
+#     @provide_session
+#     def load_user(self, userid, session=None):
+#         if not userid or userid == 'None':
+#             return None
+#
+#         user = session.query(models.User).filter(
+#             models.User.id == int(userid)).first()
+#         return GoogleUser(user)
+#
+#     @provide_session
+#     def oauth_callback(self, session=None):
+#         log.debug('Google OAuth callback called')
+#
+#         next_url = request.args.get('state') or url_for('admin.index')
+#
+#         resp = self.google_oauth.authorized_response()
+#
+#         try:
+#             if resp is None:
+#                 raise AuthenticationError(
+#                     'Null response from Google, denying access.'
+#                 )
+#
+#             google_token = resp['access_token']
+#
+#             username, email = self.get_google_user_profile_info(google_token)
+#
+#             if not self.domain_check(email):
+#                 return redirect(url_for('airflow.noaccess'))
+#
+#         except AuthenticationError:
+#             return redirect(url_for('airflow.noaccess'))
+#
+#         user = session.query(models.User).filter(
+#             models.User.username == username).first()
+#
+#         if not user:
+#             user = models.User(
+#                 username=username,
+#                 email=email,
+#                 is_superuser=False)
+#
+#         session.merge(user)
+#         session.commit()
+#         login_user(GoogleUser(user))
+#         session.commit()
+#
+#         return redirect(next_url)
 
 
-def login(self, request):
-    return login_manager.login(request)
+# login_manager = GoogleAuthBackend()
+#
+#
+# def login(self, request):  # pylint:disable=unused-argument,redefined-outer-name
+#     return login_manager.login(request)
```

Comment:
I commented all this because I couldn't find any usages of classes and methods from the file. @mik-laj what do you think?